### ISSUE TITLE
Request for modification to alarm settings, separate from in-range settings.

### ISSUE THREAD

**Issue by PhilaBurger** [creepymonster/GlucoseDirect#512]
I would like to request Glucose level alarm levels that would be separate from the current Low and High settings.

My assumption is that the current Low and High would be used for one's target range...say, for example, 80-160mg/dL.

With the current setting options, if one has the **Alarm Glucose Notification** enabled, one would get an alarm for low glucose at 79mg/dL or for high glucose at 181mg/dL.

Given that one may frequently go out of range, but not so severely that it would require an alarm, my suggestion is to add another pair of settings for alarms, themselves, that could be set for the same or different levels, similar to the settings in the actual Libre app.

In this way, the user can set their target range **and** an alarm range wherein they would be able to A) be alerted if they've reached or exceeded a critical level and B) see when they were in range, out of range or in their perceived danger zones.

**Comment by JakubSliwinski85pl** [comment]
I agree with the above, this is a needed option.  + Additionally, please add the function of rising alarm (low+high) where the sound signal will be preceded by e.g. two or three cycles of vibration without sound signal and then the sound alarm is additionally activated.
